pkgs/by-name/br/brainflow/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "brainflow";
+  version = "5.11.0";
+
+  src = fetchFromGitHub {
+    owner = "brainflow-dev";
+    repo = "brainflow";
+    rev = version;
+    hash = "sha256-mV/8mEAMsc1VyFmNinX4KUHswuspZL0l0iZCGULdXB8=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  postPatch = ''
+    for file in \
+      "cpp_package/build.cmake" \
+      "src/ml/build.cmake" \
+      "src/board_controller/openbci/ganglion_bglib/build.cmake" \
+      "src/utils/bluetooth/build.cmake" \
+      "src/board_controller/build.cmake" \
+      "src/board_controller/neuromd/brainbit_bglib/build.cmake" \
+      "src/board_controller/muse/muse_bglib/build.cmake" \
+      "src/data_handler/build.cmake"; do
```

Comment:
Does
```suggestion
    for file in **/build.cmake; do
```
work?

Reply:
not completely, it produces the output below

```
/nix/store/ainnpdma6klsb6wky4phj1n1z2zf0bqa-brainflow-5.11.0
 ├──inc
 │  ├──board_controller.h
 │  ├──board_info_getter.h
 │  ├──brainflow_array.h
 │  ├──brainflow_constants.h
 │  ├──brainflow_exception.h
 │  ├──brainflow_input_params.h
 │  ├──brainflow_model_params.h
 │  ├──data_handler.h
 │  ├──json.hpp
 │  ├──ml_module.h
 │  └──shared_export.h
 ├──include
 │  ├──board_shim.h
 │  ├──brainflow_array.h
 │  ├──brainflow_exception.h
 │  ├──data_filter.h
 │  └──ml_model.h
...
```

Reply:
`find`/`sed` works, do you want to go with that?

```
    find . -type f -name 'build.cmake' -exec \
      sed -i 's/DESTINATION inc/DESTINATION include/g' {} \;
```

Reply:
I like the more explicit approach. Keep it.